ckanext/datagovtheme/public/css/datagovtheme.css

```diff
@@ -18,6 +18,13 @@
     background-color: transparent;
     border: 0;
 }
+.hide {
```

Comment:
anyone know the reason for this alt CSS file?

Reply:
Apparently CKAN CSS messed with Data.gov theme CSS and USWDS CSS... this is where we use it @btylerburton 
```html
ckanext/datagovtheme/templates/header-site-usage.html:3:<a href="#content" class="hide">{{ _('Skip to content') }}</a>
ckanext/datagovtheme/templates/header-site-usage.html:18:                        <label class="hide" for="search-header">Search for:</label>
ckanext/datagovtheme/templates/package/search.html:15:          <label for="search-big" class="hide">{{ _('Search datasets') }}</label>
ckanext/datagovtheme/templates/package/search.html:124:        <label for="field-location" class="hide">{{ _('Filter by location') }}</label>
ckanext/datagovtheme/templates/header.html:63:<a href="#content" class="hide">{{ _('Skip to content') }}</a>
ckanext/datagovtheme/templates/header.html:104:                            <li class="usa-nav__primary-item desktop-hide">
```